source/features/bugs-tab.tsx

```diff
@@ -24,14 +24,46 @@ async function highlightBugsTabOnIssuePage(): Promise<void | false> {
 	issuesTab.removeAttribute('aria-current');
 }
 
+const bugLabelCacheKey = (): string => 'bugs-label:' + getRepo()!.nameWithOwner;
+const bugLabel = async (): Promise<string | undefined> => cache.get<string>(bugLabelCacheKey());
```

Comment:
Let's make it clear that these are getters

```suggestion
const getBugLabelCacheKey = (): string => 'bugs-label:' + getRepo()!.nameWithOwner;
const getBugLabel = async (): Promise<string | undefined> => cache.get<string>(getBugLabelCacheKey());
```